src/bitmessagekivy/baseclass/addressbook_widgets.py

```diff
@@ -19,12 +19,14 @@ def __init__(self):
     @staticmethod
     def default_label_when_empty():
         """This function returns default message while no address is there."""
+        no_address_found = "No contact found yet......"
```

Comment:
can you make it one of:
- class variable
- constanst
- module variable (this is probably the best choice)